explorer/ast/member.h

```diff
@@ -33,16 +44,30 @@ class Member {
  public:
   explicit Member(Nonnull<const Declaration*> declaration);
   explicit Member(Nonnull<const NamedValue*> struct_member);
+  explicit Member(Nonnull<const IndexedValue*> tuple_member);
+
+  // Return whether the member's name matches `name`.
+  auto IsNamed(std::string_view name) const -> bool;
+  // Prints the Member
+  void Print(llvm::raw_ostream& out) const;
+
+  // Return whether the member is positional, i.e. has an index.
+  auto hasPosition() const -> bool;
+  // Return whether the member is named, i.e. has a name.
+  auto hasName() const -> bool;
```

Comment:
Names should either be accessor-named if trivial (has_name) or normal function named (HasName)

Reply:
Git it. Done